filebeat/docs/autodiscover-docker-config.asciidoc

```diff
@@ -0,0 +1,39 @@
+Filebeat supports templates for both prospectors and modules.
+
+["source","yaml",subs="attributes"]
+-------------------------------------------------------------------------------------
+filebeat.autodiscover:
+  providers:
+    - type: docker
+      templates:
+        - condition:
+            equals:
+              docker.container.image: redis
+          config:
+            - type: docker
+              container.ids:
+                - "${data.docker.container.id}"
+              exclude_lines: ["^\\s+[\\-`('.|_]"]  # drop asciiart lines
+-------------------------------------------------------------------------------------
+
+This will launch `redis` prospector for all containers running a image with `redis` in its name.
```

Comment:
By "this" you mean the above? Seems to start the docker prospector for Redis logs, not the redis prospector. Or the below which seems to start the `redis` module :).

Reply:
You are right :innocent: , it launches the docker prospector for the redis container, will fix that, thanks!